Packs/CortexXDR/ReleaseNotes/5_2_6.md

```diff
@@ -0,0 +1,137 @@
+
+#### Scripts
+
+##### CortexXDRAdditionalAlertInformationWidget
+
+- Updated the Docker image to: *demisto/python3:3.10.13.80014*.
+
+##### CortexXDRCloudProviderWidget
+
+- Updated the Docker image to: *demisto/python3:3.10.13.80014*.
+
+##### CortexXDRIdentityInformationWidget
+
+- Updated the Docker image to: *demisto/python3:3.10.13.80014*.
+
+##### CortexXDRInvestigationVerdict
+
+- Updated the Docker image to: *demisto/python3:3.10.13.80014*.
+
+##### CortexXDRRemediationActionsWidget
+
+- Updated the Docker image to: *demisto/python3:3.10.13.80014*.
+
+##### DBotGroupXDRIncidents
+
+- Updated the Docker image to: *demisto/python3:3.10.13.80014*.
+
+##### EntryWidgetNumberHostsXDR
+
+- Updated the Docker image to: *demisto/python3:3.10.13.80014*.
+
+##### EntryWidgetNumberRegionsXCLOUD
+
+- Updated the Docker image to: *demisto/python3:3.10.13.80014*.
+
+##### EntryWidgetNumberResourcesXCLOUD
+
+- Updated the Docker image to: *demisto/python3:3.10.13.80014*.
+
+##### EntryWidgetNumberUsersXDR
+
+- Updated the Docker image to: *demisto/python3:3.10.13.80014*.
+
+##### EntryWidgetPieAlertsXDR
+
+- Updated the Docker image to: *demisto/python3:3.10.13.80014*.
+
+##### XCloudRegionsPieWidget
+
+- Updated the Docker image to: *demisto/python3:3.10.13.80014*.
+
+##### XCloudResourcesPieWidget
+
+- Updated the Docker image to: *demisto/python3:3.10.13.80014*.
+
+##### XDRConnectedEndpoints
+
+- Updated the Docker image to: *demisto/python3:3.10.13.80014*.
+
+##### XDRDisconnectedEndpoints
+
+- Updated the Docker image to: *demisto/python3:3.10.13.80014*.
+
+
+#### Integrations
+
+##### Cortex XDR - IOC
+
+- Updated the Docker image to: *demisto/python3:3.10.13.80014*.
+
+##### Cortex XDR - XQL Query Engine
+
+- Updated the Docker image to: *demisto/python3:3.10.13.80014*.
+
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+
+- Added the *public_ip_list* argument for the ***xdr-get-endpoints*** command.
+- Updated the Docker image to: *demisto/python3:3.10.13.80014*.
+
+
+#### Playbooks
+
+##### Cortex XDR - First SSO Access
+
+- Updated the marketplaces to xsoar only.
```

Comment:
Not really sure what this should be, but perhaps it should be changed to the following.  And if this is correct, please change everywhere. (And remove the bullet (-).
Updated in Marketplace to Cortex XSOAR only.